realm/realm-library/src/main/java/io/realm/RealmMap.java

```diff
@@ -355,7 +355,7 @@ public V put(K key, @Nullable V value) {
             return putInternal(key, value);
         }
 
-        protected void checkValidKey(@Nullable K key) {
+        protected void checkValidKey(K key) {
```

Comment:
Same. Have we moved things around so that a package-info doesn't set the default anymore?